docs/v2-roadmap.rst

```diff
@@ -1,83 +1,226 @@
-v2.0 Roadmap
-============
+v2.0 Migration Guide
+====================
 
-.. important::
+**urllib3 v2.0 is now available!** Read below for how to get started and what is contained in the new major release.
 
-   We're seeking `sponsors and supporters for urllib3 v2.0 on Open Collective <https://github.com/sponsors/urllib3>`_.
-   There's a lot of work to be done for our small team and we want to make sure
-   development can get completed on-time while also fairly compensating contributors
-   for the additional effort required for a large release like ``v2.0``.
+**🚀 Migrating from 1.x to 2.0**
+--------------------------------
+
+We're maintaining **functional API compatibility for most users** to make the
+migration an easy choice for almost everyone. Most changes are either to default
+configurations, supported Python versions, or internal implementation details.
+So unless you're in a specific situation you should notice no changes! 🎉
+
+.. note::
+
+  If you have difficulty migrating to v2.0 or following this guide
+  you can `open an issue on GitHub <https://github.com/urllib3/urllib3/issues>`_
+  or reach out in `our community Discord channel <https://discord.gg/urllib3>`_.
+
+
+Timeline for deprecations and breaking changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The 2.x initial release schedule will look like this:
+
+* urllib3 ``v2.0.0-alpha1`` will be released in November 2022. This release
+  contains **minor breaking changes and deprecation warnings for other breaking changes**.
+  There may be other pre-releases to address fixes before v2.0.0 is released.
+* urllib3 ``v2.0.0`` will be released in early 2023 after some initial integration testing
+  against dependent packages and fixing of bug reports.
+* urllib3 ``v2.1.0`` will be released in the summer of 2023 with **all breaking changes
+  being warned about in v2.0.0**.
+
+.. warning::
+
+  Please take the ``DeprecationWarnings`` you receive when migrating from v1.x to v2.0 seriously
+  as they will become errors after 2.1.0 is released.
+
+
+What are the important changes?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Here's a short summary of which changes in urllib3 v2.0 are most important:
+
+- Python version must be **3.7 or later** (previously supported Python 2.7, 3.5, and 3.6).
+- Removed support for non-OpenSSL TLS libraries (like LibreSSL and wolfSSL).
+- Removed support for OpenSSL versions older than 1.1.1.
+- Removed support for Python implementations that aren't CPython or PyPy3 (previously supported Google App Engine, Jython).
+- Removed ``urllib3.contrib.pyopenssl``, ``urllib3.contrib.securetransport``, and ``urllib3.contrib.ntlmpool`` modules.
+- Removed the ``urllib3[secure]`` extra.
```

Comment:
I think those are still in main and will only go in 2.1.0? (except ntlmpool, this one is gone already)